src/webview_provider/docsEditPanel.ts

```diff
@@ -383,9 +384,26 @@ export class DocsEditViewPanel implements WebviewViewProvider {
     if (!data.length) {
       return;
     }
-    return {
-      tests: data,
-    };
+    const dbtVersion = project.getDBTVersion();
+    if (dbtVersion && dbtVersion[0] >= 1 && dbtVersion[1] >= 8) {
+      if (
+        existingColumn &&
+        existingColumn.name === columnNameFromWebview &&
```

Comment:
i believe this will always be true

Reply:
no actually, all the columns with tests are passed here, even the ones that had no additional changes